.github/workflows/sync-docs-full.yml

```
@@ -0,0 +1,87 @@
+      - name: Send to Agentuity
+        run: |
+          echo "About to sync these files:"
+          jq -r '.changed[].path' payload.json
+          echo -e "\nWill first remove these paths:"
+          jq -r '.removed[]' payload.json
+          
+          # Uncomment to actually send
+          # curl https://agentuity.ai/webhook/f61d5ce9d6ed85695cc992c55ccdc2a6 \
+          #   -X POST \
+          #   -H "Content-Type: application/json" \
+          #   -d @payload.json 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)